### ISSUE TITLE
configure Jetty plugin to work for both jetty:run and 'integration' profile

### ISSUE THREAD

**Issue by justinhrobbins** [justinhrobbins/FlashCards_App#26]
also remove contexthandlers from 'integration' profile which currently throw exception and are not needed for integration tests

maybe define jetty plugin in both a 'default' profile and 'integration'

use <pluginManagement> to define the plugin only once?

an example of <pluginManagement> here:
https://github.com/mike-ensor/multi-module-site-doc-project/blob/master/pom.xml
